src/components/KeyboardShortcuts/index.spec.tsx

```diff
@@ -129,6 +130,31 @@ describe(__filename, () => {
     expect(root.find(`.${styles.KeyboardShortcuts}`)).toHaveLength(1);
   });
 
+  it('does not render a description for alias keys', () => {
+    const root = render();
+
+    const aliasKeys = Object.keys(supportedKeys).filter(
+      (key) => supportedKeys[key] === null,
+    );
+
+    for (const key of aliasKeys) {
+      // The expression below tries to find all react components with $key as
```

Comment:
I found this comment confusing. How about this?

"The expression below will look for any JSX rendered with an alias key, such as a `<kbd>` tag."